docs/src/app/tabs.jsx

```diff
@@ -0,0 +1,95 @@
+import { browserHistory, withRouter } from 'react-router';
+import {
+    findIndex,
+    map,
+    remove,
+} from 'lodash';
+import { SectionalTabs } from 'react-cm-ui';
+import PropTypes from 'prop-types';
+import React, { useCallback, useEffect, useState } from 'react';
```

Comment:
Can we place each imported element in a new line after the `React` import?
```
import React, {
    useCallback,
    useEffect,
    useState,
} from 'react';
```

Reply:
Fixed.